src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -1232,6 +1238,12 @@ public override void VisitArrayInitializer(IArrayInitializerOperation operation)
         public override void VisitSimpleAssignment(ISimpleAssignmentOperation operation)
         {
             LogString(nameof(ISimpleAssignmentOperation));
+
+            if (operation.IsRef)
+            {
+                LogString(" (IsRef)");
```

Comment:
I didn't see which test exercises this.

Reply:
> I didn't see which test exercises this.

None at the moment. Only lowered tree can have nodes like this. Once ref reassignment feature is supported, then we can test this API. 

---
In reply to: [148105993](https://github.com/dotnet/roslyn/pull/22933#discussion_r148105993) [](ancestors = 148105993)